packages/@aws-cdk/aws-servicecatalog/test/portfolio.test.ts

```diff
@@ -331,49 +331,34 @@ describe('portfolio associations and product constraints', () => {
     Template.fromStack(stack).resourceCountIs('AWS::ServiceCatalog::TagOptionAssociation', 3);
   }),
 
-  test('adding identical tag options to portfolio is idempotent', () => {
```

Comment:
this test (and functionality) is no longer valid.

Reply:
This is the interesting point that the original issue brought up, that I don't exactly know best way to handle this to fail faster than deploy time.  Tag Options are globally unique.  Moving towards the construct approach, then I don't you shouldn't be able to even make `tagOptions2`.  I know that we can check the constructs in a stack, but can we check all of the apps stack to see if same `tagOption` has already been made in an efficient manner?